website/templates/public/pages/getting_started.mako

```diff
@@ -110,7 +110,7 @@
         </div>
 
 
-        <div  id="structure" class="row col-sm-12 col-md-12" style="padding-top: 40px;">
+        <div  id="structure" class="row col-sm-12 col-md-12" style="padding-top: 40px">
```

Comment:
Can you add the semicolons back in/make sure there's no diff with this file? It's best to include only the changes relevant to the issue that you are solving.